packages/server/lib/cloud/api.ts

```diff
@@ -244,11 +256,11 @@ module.exports = {
     }
   },
 
+  // TODO: i think we can remove this function
```

Comment:
I don't think we can without reworking a lot of tests

Reply:
i basically already did that, haha... but its nbd, just leaving myself a note if I get it for free at the end as a reminder